### ISSUE TITLE
webpack dev server live reloading not working when i set target:["web","es5"] in webpack config.

### ISSUE THREAD

**Issue by irfanit93** [webpack/webpack-dev-server#2788]
- Operating System:
- Node Version:
- NPM Version:
- webpack Version:
- webpack-dev-server Version:
- Browser:

- [ ] This is a **bug**
- [ ] This is a **modification** request

### Code

```js
// webpack.config.js
```

```js
// additional code, remove if not needed.
```

**Comment by evilebottnawi** [comment]
Duplicate already reported

**Comment by evilebottnawi** [comment]
Release will be in near future, right now use `target: 'web'`

**Comment by irfanit93** [comment]
Okay cool

**Comment by rsxdalv** [comment]
Is the fix in v4 or still under v3.11?

**Comment by alexander-akait** [comment]
fixed in v4
